?  How to Update from CVS.txt
? build-release
? mac_native_splines.patch
? mac_print_max_resolution.patch
? rebuild-wxw.sh
? contrib/samples/deprecated/proplist/.DS_Store
? contrib/samples/deprecated/resource/.DS_Store
? contrib/samples/deprecated/treelay/.DS_Store
? demos/bombs/.DS_Store
? demos/dbbrowse/.DS_Store
? demos/forty/.DS_Store
? demos/fractal/.DS_Store
? demos/life/.DS_Store
? demos/poem/.DS_Store
? include/.DS_Store
? include/wx/.DS_Store
? include/wx/setup.h
? include/wx/mac/.DS_Store
? include/wx/mac/carbon/.DS_Store
? samples/.DS_Store
? samples/access/.DS_Store
? samples/artprov/.DS_Store
? samples/calendar/.DS_Store
? samples/caret/.DS_Store
? samples/checklst/.DS_Store
? samples/config/.DS_Store
? samples/controls/.DS_Store
? samples/db/.DS_Store
? samples/dialogs/.DS_Store
? samples/dnd/.DS_Store
? samples/docview/.DS_Store
? samples/docview/Carbon_MachO(2.5.3).app
? samples/docview/Carbon_MachO.app
? samples/docview/Carbon_MachODebug(2.5.3).app
? samples/docview/Carbon_MachODebug.app
? samples/docview/docview.plc
? samples/docview/docviewM8.mcp
? samples/docvwmdi/.DS_Store
? samples/dragimag/.DS_Store
? samples/drawing/.DS_Store
? samples/dynamic/.DS_Store
? samples/erase/.DS_Store
? samples/event/.DS_Store
? samples/exec/.DS_Store
? samples/font/.DS_Store
? samples/grid/.DS_Store
? samples/help/.DS_Store
? samples/html/about/.DS_Store
? samples/html/help/.DS_Store
? samples/html/helpview/.DS_Store
? samples/html/printing/.DS_Store
? samples/html/test/.DS_Store
? samples/html/virtual/.DS_Store
? samples/html/widget/.DS_Store
? samples/html/zip/.DS_Store
? samples/image/.DS_Store
? samples/internat/.DS_Store
? samples/joytest/.DS_Store
? samples/keyboard/.DS_Store
? samples/layout/.DS_Store
? samples/listbox/.DS_Store
? samples/listctrl/.DS_Store
? samples/mdi/.DS_Store
? samples/mediaplayer/.DS_Store
? samples/menu/.DS_Store
? samples/minifram/.DS_Store
? samples/minimal/.DS_Store
? samples/minimal/Carbon_MachO.app
? samples/minimal/Carbon_MachODebug.app
? samples/minimal/Carbon_MachO_Shlb.app
? samples/minimal/Carbon_MachO_Shlb_Debug.app
? samples/minimal/minimalM8.mcp
? samples/notebook/.DS_Store
? samples/opengl/cube/.DS_Store
? samples/opengl/isosurf/.DS_Store
? samples/opengl/penguin/.DS_Store
? samples/ownerdrw/.DS_Store
? samples/png/.DS_Store
? samples/printing/.DS_Store
? samples/printing/printing_original.cpp
? samples/propsize/.DS_Store
? samples/regtest/.DS_Store
? samples/richedit/.DS_Store
? samples/rotate/.DS_Store
? samples/sashtest/.DS_Store
? samples/scroll/.DS_Store
? samples/scrollsub/.DS_Store
? samples/shaped/.DS_Store
? samples/splitter/.DS_Store
? samples/statbar/.DS_Store
? samples/taskbar/.DS_Store
? samples/text/.DS_Store
? samples/thread/.DS_Store
? samples/toolbar/.DS_Store
? samples/treectrl/.DS_Store
? samples/typetest/.DS_Store
? samples/validate/.DS_Store
? samples/widgets/.DS_Store
? samples/wizard/.DS_Store
? src/wxWindowsM8 Data
? src/wxWindowsM8.mcp
? src/wxWindowsM9 Data
? src/wxWindowsM9.mcp
? src/common/.DS_Store
? src/jpeg/jpegM8.mcp
? src/mac/.DS_Store
? src/mac/carbon/.DS_Store
? src/png/pngM8.mcp
? src/regex/.DS_Store
? src/regex/regexM8 Data
? src/regex/regexM8.mcp
? src/tiff/tiffM8.mcp
? src/wxWindows.xcodeproj/rjlang.mode1
? src/zlib/zlibM8.mcp
? utils/HelpGen/src/.DS_Store
? utils/helpview/src/.DS_Store
? utils/tex2rtf/src/.DS_Store
Index: include/wx/mac/carbon/chkconf.h
===================================================================
RCS file: /pack/cvsroots/wxwidgets/wxWidgets/include/wx/mac/carbon/chkconf.h,v
retrieving revision 1.7
diff -b -u -2 -r1.7 chkconf.h
--- include/wx/mac/carbon/chkconf.h	2005/07/27 15:02:04	1.7
+++ include/wx/mac/carbon/chkconf.h	2005/09/28 04:05:30
@@ -32,5 +32,5 @@
 
 #ifndef wxMAC_USE_CORE_GRAPHICS
-    #define wxMAC_USE_CORE_GRAPHICS 0
+    #define wxMAC_USE_CORE_GRAPHICS 1
 #endif
 
Index: src/mac/carbon/printmac.cpp
===================================================================
RCS file: /pack/cvsroots/wxwidgets/wxWidgets/src/mac/carbon/printmac.cpp,v
retrieving revision 1.32
diff -b -u -2 -r1.32 printmac.cpp
--- src/mac/carbon/printmac.cpp	2005/09/23 12:54:10	1.32
+++ src/mac/carbon/printmac.cpp	2005/09/28 04:05:41
@@ -38,4 +38,10 @@
 #include <stdlib.h>
 
+#ifndef wxMAC_PRINT_MAX_RESOLUTION
+  // 0 = print at 72 dpi (default)
+  // 1 = print at highest resolution supported by printer
+  #define wxMAC_PRINT_MAX_RESOLUTION 0
+#endif // wxMAC_PRINT_MAX_RESOLUTION
+
 IMPLEMENT_DYNAMIC_CLASS(wxMacCarbonPrintData, wxPrintNativeDataBase)
 IMPLEMENT_DYNAMIC_CLASS(wxMacPrinter, wxPrinterBase)
@@ -151,4 +157,13 @@
     // PMQualityMode not yet accessible via API
     // todo paperSize
+    
+#if wxMAC_PRINT_MAX_RESOLUTION
+    PMResolution res;
+    PMPrinter printer;
+    PMTag tag = kPMMaxSquareResolution;
+    PMSessionGetCurrentPrinter(m_macPrintSession, &printer);
+    PMPrinterGetPrinterResolution(printer, tag, &res);
+    PMSetResolution((PMPageFormat) m_macPageFormat, &res);
+#endif
     return true ;
 }
@@ -344,5 +359,14 @@
     
     printout->SetPPIScreen(72, 72);
+    
+#if wxMAC_PRINT_MAX_RESOLUTION
+    PMResolution res;
+    wxMacCarbonPrintData* nativeData = (wxMacCarbonPrintData*)
+      (m_printDialogData.GetPrintData().GetNativeData());
+    PMGetResolution((PMPageFormat) (nativeData->m_macPageFormat), &res);
+    printout->SetPPIPrinter(int(res.hRes), int(res.vRes));
+#else
     printout->SetPPIPrinter(72, 72);
+#endif
     
     // Set printout parameters  

 	  	 
